src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import { getCommonProps } from '../../utils';
+import { px } from '../../utils';
+
+import Select from '../Select/Select';
+import Button from '../Button/Button';
+import chevronRight from '../../assets/chevronRight.svg';
+
+export interface PaginationProps {
```

Comment:
this should extend the underlying element `ComponentProps` being rendered as the root-element.  Then any non-specified props should be passed through